apps/sequencer/src/feeds/votes_result_sender.rs

```diff
@@ -228,7 +228,7 @@ async fn try_send_aggregation_consensus_trigger_to_reporters(
             calldata: hex::encode(serialized_updates),
             updates: updates.updates,
             feeds_rounds,
-            proofs: updates.proofs,
+            proofs: HashMap::new(), // No need to propagate signatures; we are currently using deviation-based validation.
```

Comment:
Consider deleting the unused struct field.

```suggestion
```

Reply:
Addressed in [PR-1153](https://github.com/blocksense-network/blocksense/pull/1153)